tests/ResourceSyncingTest.php

```diff
@@ -537,10 +761,37 @@ function creatingResourceInTenantDatabaseCreatesAndMapInCentralDatabase()
     expect(ResourceUser::first()->role)->toBe('commenter');
 }
 
-function migrateTenantsResource()
+function creareTenantsAndRunMigrations(): array
```

Comment:
Add `/**` comments above all of the helpers you created to explain the point of them. In general, test helpers are pretty confusing unless they're super simple, so they should always be explained well.

Reply:
Added the comments on each helper method.